packages/react-composites/src/composites/CallComposite/adapter/AzureCommunicationCallAdapter.ts

```diff
@@ -580,3 +614,8 @@ export const createAzureCommunicationCallAdapterFromClient = async (
 const isCallError = (e: Error): e is CallError => {
   return e['target'] !== undefined && e['innerError'] !== undefined;
 };
+
+/* @conditional-compile-remove-from(stable) TEAMS_ADHOC_CALLING */
+const isAdhocCall = (callLocator: CallAdapterLocator): callLocator is CallParticipantLocator => {
```

Comment:
Mark is at @private so we don't accidentally export it.

Reply:
For small helper fns not exported out of the file we typically don't require any modifier tags